.travis.yml

```
@@ -0,0 +1,31 @@
+services:
+- docker
+script:
+- docker build --target test --tag todo-app:test .
+- docker run --env-file .env.test todo-app:test tests
+- docker run
+  -e SECRET_KEY=$SECRET_KEY
+  -e TRELLO_ACCOUNT_KEY=$TRELLO_ACCOUNT_KEY
+  -e TRELLO_SECRET_KEY=$TRELLO_SECRET_KEY
+  -e BOARD_ID=$BOARD_ID
+  -e TODO_LIST_ID=$TODO_LIST_ID
+  -e IN_PROGRESS_LIST_ID=$IN_PROGRESS_LIST_ID
+  -e DONE_LIST_ID=$DONE_LIST_ID
+  todo-app:test tests_e2e
```

Minor (and potentially irrelevant given my comment elsewhere about Travis vs Github Actions) but it's worth noting there's actually a shorthand syntax for Docker if the environment variable names match inside/outside the container:

```suggestion
- docker run
  -e SECRET_KEY
  -e TRELLO_ACCOUNT_KEY
  -e TRELLO_SECRET_KEY
  -e BOARD_ID
  -e TODO_LIST_ID
  -e IN_PROGRESS_LIST_ID
  -e DONE_LIST_ID
  todo-app:test tests_e2e
```